crypto/hash/auth.c

```diff
@@ -177,14 +176,12 @@ srtp_auth_type_test (const srtp_auth_type_t *at, const srtp_auth_test_case_t *te
     return srtp_err_status_ok;
 }
 
-
 /*
  * auth_type_self_test(at) performs srtp_auth_type_test on at's internal
```

Comment:
Historical, but since you're here: in the comment, we should have srtp_ prefixing the name of the function.